scripts/travis/before_build.sh

```diff
@@ -39,6 +39,20 @@ function runGoLint() {
     return 1
 }
 
+function runCheckLicense() {
```

Comment:
I'm not sure why you would need this change here. The changes in `check_license.sh` seems to be enough. ( unless you refer to the writing to the stderr, where I'm not a big fan of.
( i.e. it prevents capturing the output of the script by just storing all the stdout into a file )